test/data/https-everywhere-data/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "HTTPS Everywhere updater extension",
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAvn9zSMjTmhkQyrZu5UdN350nPqLoSeCYngcC7yDFwaUHjoBQXCZqGeDC69ciCQ2mlRhcV2nxXqlUDkiC6+7m651nI+gi4oVqHagc7EFUyGA0yuIk7qIMvCBdH7wbET27de0rzbRzRht9EKzEjIhCBtoPnmyrO/8qPrH4XR4cPfnFPuJssBBxC1B35H7rh0Br9qePhPDDe9OjyqYxPuio+YcC9obL4g5krVrfrlKLfFNpIewUcJyBpSlCgfxEyEhgDkK9cILTMUi5vC7GxS3POtZqgfRg8Da4i+NwmjQqrz0JFtPMMSyUnmeMj+mSOL4xZVWr8fU2/GOCXs9gczDpJwIDAQAB",
+  "manifest_version": 2,
+  "name": "HTTPS Everywhere updater",
+  "version": "1.0.0"
+}
```

Comment:
Would you mind also doing a test for tracking-protection at this point? Ask Serg if he can give you a test case example that will be blocked by tracking protection library but not the ad-block library.